journalbeat/tests/test.sh

```diff
@@ -1,32 +1,31 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
 
-TESTDIR="$(dirname "${0}")"
-PLANDIR="$(dirname "${TESTDIR}")"
-SKIPBUILD=${SKIPBUILD:-0}
+set -euo pipefail
 
-hab pkg install --binlink core/bats
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
 
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+hab pkg install core/bats --binlink
 hab pkg install core/busybox-static
 hab pkg binlink core/busybox-static ps
 hab pkg binlink core/busybox-static wc
+hab pkg install "${TEST_PKG_IDENT}"
 
-source "${PLANDIR}/plan.sh"
-
-if [ "${SKIPBUILD}" -eq 0 ]; then
-    # Unload the service if its already loaded.
-    hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+hab svc load "${TEST_PKG_IDENT}"
```

Comment:
Before loading the service, please add something to ensure the supervisor is running.  See my [comment](https://github.com/habitat-sh/core-plans/pull/2601#issuecomment-498660592)